### ISSUE TITLE
【论文复现营】关于30系列显卡计算结果误差比较大的问题

### ISSUE THREAD

**Issue by jstzwjr** [PaddlePaddle/X2Paddle#580]
定义了一个卷积层，weight全为1，bias为None，输入数据为随机。
paddle的卷积核输出结果与torch相差比较大，但是切换的cpu或者使用2080ti计算时，误差就很小。
环境：cuda11.2 pytorch1.7.0 paddle2.0.2

import numpy as np
import torch
import paddle
from paddle.nn.initializer import Assign

# paddle.set_device('cpu')
paddle_x = np.random.randn(1,64,56,56).astype('float32')
torch_x = paddle_x.copy()
paddle_w = np.ones((128,64,3,3)).astype('float32')
torch_w = paddle_w.copy()
torch_conv = torch.nn.Conv2d(in_channels=64,out_channels=128,kernel_size=3,stride=2,bias=False)
paddle_conv = paddle.nn.Conv2D(in_channels=64,out_channels=128,kernel_size=3,stride=2,bias_attr=None)
torch_conv.weight = torch.nn.Parameter(torch.from_numpy(torch_w))
parameter = paddle_conv.create_parameter(shape=paddle_w.shape,default_initializer=Assign(paddle_w))
paddle_conv.add_parameter("weight",parameter)

torch_y = torch_conv(torch.from_numpy(torch_x)).cpu().detach().numpy()
paddle_y = paddle_conv(paddle.to_tensor(paddle_x)).numpy()
print(np.max(np.abs(torch_y-paddle_y)))

**Comment by SunAhong1993** [comment]
这边我们查看下原因，有结果将及时回复您~

**Comment by SunAhong1993** [comment]
我们目前未在这一系列的机器上进行对齐，近期将进行对齐工作，建议你们在1080和2080上实验，该部分机器我们已经对齐过。
